src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/types/SinglePaymentResponse.java

```diff
@@ -0,0 +1,45 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice.types;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * SinglePaymentResponse 
+ * 
+ * @author 176899
+ *
+ */
+@XmlRootElement
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SinglePaymentResponse {
+
+	private String respMsg;
+	private int respCode;
+	private String respValue;
+	
+	public SinglePaymentResponse(String respMsg, int respCode, String respValue) {
+		this.respMsg = respMsg;
+		this.respCode = respCode; 
+		this.respValue = respValue; 
+	}
+	
+	public String getRespMsg() {
+		return respMsg;
+	}
+	public void setRespMsg(String respMsg) {
```

Comment:
Can we remove the setters since we are using the constructor ?

Reply:
We could, but the setter gives us a way to change an objects value at runtime.